client/src/js/services/FilterService.depricated.js

```diff
@@ -0,0 +1,53 @@
+angular.module('bhima.services')
+  .service('DepricatedFilterService', DepricatedFilterService);
```

Comment:
This is just further incentive to get the files/ code removed from the repository.